experimental/FTheoryTools/src/LiteratureModels/Models/model1408_4808-11-WSF.json

```diff
@@ -41,8 +41,8 @@
     "model_descriptors": {
         "type": "weierstrass",
         "description": "F-theory weierstrass model dual to hypersurface model with fiber ambient space F_11",
-        "gauge_algebra": ["su3","su2","u1"],
-        "global_gauge_quotients": [["Z3", [1, 3]], ["Z2", [2, 3]]]
+        "gauge_algebra": ["su(3)","su(2)","u(1)"],
+        "global_gauge_quotients": [["diagonal_matrix(root_of_unity(C,3),3)"], ["-identity_matrix(C,2)"], ["diagonal_matrix(root_of_unity(C,6,-1),1)"]]
```

Comment:
I suppose this another place holder? For I should have expected some groups or group homomorphisms here. Could you state what is currently missing in OSCAR? Matrix Lie groups perse, or rather "just" some necessary homomorphisms?

Reply:
In general Matrix Lie groups over the complex numbers are missing. This is an attempted workaround due to that fact. As I understand here for each component of the gauge algebra I've listed the elements of the center that we are quotienting by to replicate the corresponding action of the gauge quotient on the center of the lie algebra. Together this should be enough to reconstruct the gauge group, but I've not yet tried.

Reply:
I see. Thank you. We should document this, so that we still understand in a year or two what the meaning was.  And why (a.k.a. what feature is missing). Likewise, I suggest a comment why the gauge algebra is currently a vector of algebras rather than their direct sum.

Reply:
That is all fine (for now). Even if we do not want to state that the direct sum is currently WIP (why? can be changed once it is there), it would not hurt to explain for outsiders/users how to read the global gauge quotients, correct?

Reply:
Yes, documenting the notation for the quotients seems wise. As for the WIP direct sum, that seems like something for the comments? I'm fine mentioning it in the documentation though if you prefer.

Reply:
Yeah, that is probably the way to go. (I don't want @lgoettgens to feel pressured. Of course we are looking forward to this feature. But then, it is not super urgent.)

Reply:
Where should I place a comment about the notation for the quotients?

Reply:
I believe we want a method `global_gauge_quotients(m::AbstractFTheoryModel)`, similar to `gauge_algebra(m::AbstractFTheoryModel)`. Just as for `gauge_algebra(m::AbstractFTheoryModel)`, you could provide details of the implementation in the doc string/the example of this function.

Btw., those doc strings need to be added to the documentation. That is, you need to modify the corresponding files in `FTheoryTools/docs` subfolder. In the case at hand, take a look at `Oscar.jl/experimental/FTheoryTools/docs/src/generalities.md`.